jbmc/src/java_bytecode/java_bytecode_language.h

```diff
@@ -240,6 +253,9 @@ class java_bytecode_languaget:public languaget
   /// It tracks objects that should be reference-equal to each other by mapping
   /// IDs of such objects to symbols that store their values.
   std::unordered_map<std::string, object_creation_referencet> references;
+
+  /// Enables checking whether we should elaborate a method body
```

Comment:
The documentation should state what the variable should contain, alongside what its purpose is.